apps/src/templates/curriculumCatalog/no_sections_to_assign_dialog.module.scss

```diff
@@ -0,0 +1,34 @@
+.dialogueContainer {
+  height: 18rem;
+  display: flex;
+  flex-flow: column;
+
+  hr {
+    border: 1px solid #000;
+    margin: 1rem 0;
+  }
+
+  h3 {
+    margin: 0 0 0 0;
```

Comment:
This can use margin shorthand: `margin: 0` 🤓

Reply:
Ah yes, thanks!